### PR TITLE
Fix title and subtitle case

### PR DESCRIPTION
**Pull request by bappi**
Following wp core standards

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/utils/variable-products.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ async function showVariableProductTour( browser, show ) {
 	const productPageURL = 'wp-admin/post-new.php?post_type=product';
 	const addProductPage = await browser.newPage();
 
-	// Go to "Add new product" page
+	// Go to "Add New Product" page
```
Comment (rodelgc) [inline plugins/woocommerce/tests/e2e-pw/utils/variable-products.js:131]: This change looks fine with me, since it's just a comment update.

**Comment by bappi** [comment]
Hello @rodelgc, is there anything else I need to do from my end? I can see it's saying review required.

**Comment by nigeljamesstevenson** [comment]
Hi @bappi -  The checks are failing because the update requires a changelog file.

Can you generate a changelog for this PR? You can do that by using the following command and following the prompts and commiting the file it generates:

`pnpm --filter=woocommerce run changelog add`

Thanks a lot!

File: plugins/woocommerce/includes/class-wc-post-types.php [hunk 2]
```diff
@@ -458,7 +458,7 @@ public static function register_post_types() {
 							'singular_name'         => __( 'Coupon', 'woocommerce' ),
 							'menu_name'             => _x( 'Coupons', 'Admin menu name', 'woocommerce' ),
 							'add_new'               => __( 'Add coupon', 'woocommerce' ),
-							'add_new_item'          => __( 'Add new coupon', 'woocommerce' ),
+							'add_new_item'          => __( 'Add New Coupon', 'woocommerce' ),
```
Comment (nigeljamesstevenson) [inline plugins/woocommerce/includes/class-wc-post-types.php:461]: I think this should stay consistent with the other entries

File: plugins/woocommerce/includes/class-wc-post-types.php [hunk 3]
```diff
@@ -324,7 +324,7 @@ public static function register_post_types() {
 						'all_items'             => __( 'All Products', 'woocommerce' ),
 						'menu_name'             => _x( 'Products', 'Admin menu name', 'woocommerce' ),
 						'add_new'               => __( 'Add New', 'woocommerce' ),
-						'add_new_item'          => __( 'Add new product', 'woocommerce' ),
+						'add_new_item'          => __( 'Add New Product', 'woocommerce' ),
```
Comment (nigeljamesstevenson) [inline plugins/woocommerce/includes/class-wc-post-types.php:327]: I think this should stay consistent with the other entries

**Comment by nigeljamesstevenson** [comment]
If someone from @woocommerce/proton could take a peek just to be sure before merging, that would be great. 

The changes are capitalizations but it is worth considering other impacts in case of possible dependencies on the original text case. 

There are also a couple of changes in `plugins/woocommerce/includes/class-wc-post-types.php` that are not consistent with the other capitalizations.

e.g. `Add new coupon` is being changed to `Add New Coupon` but `Add coupon` has not been updated in a similar manner (and various others)

**Comment by coreymckrill** [comment]
It looks like all of these changes are moving a string from sentence case to title case. I think there are two issues with this:

1. It will negate a lot of translations. Even though only the case of a letter is changing, I believe the GlotPress translation system will interpret this as a different string and require a new translation, for every locale.
2. Our [localization guidelines](https://github.com/woocommerce/woocommerce/wiki/String-localisation-guidelines) suggest that we should favor sentence case.

As such, I don't think we should merge this.

**Comment by bappi** [comment]
I just wanted to make all Strings in settings in the same case, if the guideline says we should follow sentence case, then we should do that. I can submit another PR for that. I just wanted to be clear about this, because now it has mixed cases and it really looks bad and inconsistent.

**Comment by coreymckrill** [comment]
@bappi There are for sure a few places in the Admin UI that are using title case when they should probably be using sentence case. But we have to be super careful about changing translatable strings, especially when it's simply for cosmetic reasons, because we have run into issues before where a changed string unexpectedly broke a part of the UI for some locales ([example](https://github.com/woocommerce/woocommerce/issues/36490)).

My recommendation would be to [create an issue](https://github.com/woocommerce/woocommerce/issues/new/choose) first, listing the instances where the wrong case is used. Then, depending on how many there are, it would probably make sense to break up the changes into separate PRs either by line or by file. That way if one of the changes ends up breaking something, it's easier to revert just that change rather than a whole bunch across the whole Admin UI.

**Comment by freecookbr** [comment]
Add this function in your functions.php theme file.

// Bug correction on Adressess at My Account page
add_filter('woocommerce_account_menu_items', function($menu_links) {
    $menu_links['edit-address'] = _n('Address', 'Addresses', (1 + (int) wc_shipping_enabled()), 'domain');
    return $menu_links;
});

**Pull request closed without merging**